packages/editor-ui/src/views/SignupView.vue

```diff
@@ -113,8 +113,8 @@ export default mixins(
 		async onSubmit(values: {[key: string]: string | boolean}) {
 			try {
 				this.loading = true;
-				const inviterId = this.$route.query.inviterId.toString();
-				const inviteeId = this.$route.query.inviteeId.toString();
+				const inviterId = (!this.$route.query.inviterId || typeof this.$route.query.inviterId !== 'string') ? null : this.$route.query.inviterId;
+				const inviteeId = (!this.$route.query.inviteeId || typeof this.$route.query.inviteeId !== 'string') ? null : this.$route.query.inviteeId;
```

Comment:
this is an edge case.. maybe better to set inviterId and inviteeId as data and so we don't need to check types here.. anyways we should not continue to `signup` with these values as null